examples/operator/operator.yaml

```
@@ -5773,7 +5773,7 @@ spec:
               fieldPath: metadata.name
         - name: OPERATOR_NAME
           value: vitess-operator
-        image: planetscale/vitess-operator:v2.0.0
+        image: planetscale/vitess-operator:latest
```

I prefer to pin examples to a specific tag so the examples don't break. I've just tagged vitess-operator v2.2.0 which is compatible with vitess v8.0.x.

```suggestion
        image: planetscale/vitess-operator:v2.2.0
```